docs/networks/testnet/become-a-validator.md

```diff
@@ -0,0 +1,129 @@
+---
```

Comment:
there is maybe duplicated code here with the miannet doc - maybe we can just say -> go check mainnet doc and replace x and y.